tables/he.utb

```diff
@@ -66,21 +57,13 @@ word 'em =
 word 'tis =
 word 'twas =
 begnum # 456-1456 print number sign before number
-midnum , 2
-postpunc , 2
-word , 2
+always , 2
 decpoint . 46
 always - 36
-postpunc . 256
-word . 256
-postpunc ; 23
-word ; 23
-midnum : 25
-postpunc : 25
+always . 256
+always ; 23
 always : 25
-postpunc ! 235
 always ! 235
-midnum / 34
 always / 34
```

Comment:
Note that some rules below, like `always / 34`, `always ~ 45`, and more, have the same mapping as the definitions in en-chardefs.cti, which means that at least for forward translation they are redundant. For backward translation it is not clear whether the rules are needed or not because there are conflicting symbols (with the same dot mapping) and the tests do not cover back-translation.